lisp/ui/widgets/dbmeter.py

```diff
@@ -143,7 +183,26 @@ def paintEvent(self, e):
             )
             painter.drawPixmap(decayRect, self._pixmap, decayRect)
 
+            # Draw markings
+            x_start = meterRect.x() + meterRect.width() / 2
+            x_end = meterRect.x() + meterRect.width()
+            for mark in self._markings:
+                painter.drawLine(QPointF(x_start, mark[0]), QPointF(x_end, mark[0]))
+
```

Comment:
This should be moved before drawing the "decay indicator", or sometime you might see only the left half of the indicator

Reply:
Intentional.

Reply:
make sense, for some reason I find it a bit annoying, but probably in normal conditions is not really noticeable :+1: